src/separations.h

```diff
@@ -206,6 +206,20 @@ class Separations
   // state var.
   std::map<std::string, cyclus::toolkit::ResBuf<cyclus::Material> > streambufs;
 
+  #pragma cyclus var { \
+    "default": "separations_record_input", \
```

Comment:
Its seems like there should be a naming convention rather than switching between `lower_with_underscores` and `CapCase`